frontend/packages/core/src/NPS/banner.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+
+import type { EmojiType } from "../Assets/emojis";
+import { HappyEmoji } from "../Assets/emojis";
+import { Button } from "../button";
+import { useAppContext } from "../Contexts";
+import { FeatureOn, SimpleFeatureFlag } from "../flags";
+import Grid from "../grid";
+import Paper from "../paper";
+import styled from "../styled";
+import { Typography } from "../typography";
+
+export interface BannerFeedbackProps {
+  /**
+   * Whether this component should appear integrated into the page versus in its own container
+   * @defaultValue false
+   */
+  integrated?: boolean;
+  /**
+   * The icon to display or null if none
+   * @defaultValue <HappyEmoji />
+   */
+  icon?: EmojiType | React.ReactNode | null;
+  /**
+   * Feedback text
+   * @defaultValue "Enjoying this feature? We would like your feedback!"
+   */
+  feedbackText?: string | null;
+  /**
+   * Feedback Button Text
+   * @defaultValue "Give Feedback"
+   */
+  feedbackButtonText?: string;
+  /**
+   * Default option in dropdown to select
+   * @defaultValue ""
+   */
+  defaultOption?: string;
+}
+
+const StyledPaper = styled(Paper)({
+  padding: "0px 16px",
+});
+
+/**
+ * An NPS Feedback Banner which will open up the NPSHeader component to ask for feedback
+ */
+export const Banner = ({
+  integrated = false,
+  icon = <HappyEmoji />,
+  feedbackText = "Enjoying this feature? We would like your feedback!",
+  feedbackButtonText = "Give Feedback",
+  defaultOption,
+}: BannerFeedbackProps) => {
+  const { triggerHeaderItem } = useAppContext();
+  const banner = (
+    <Grid container direction="row" spacing={1} alignItems="center">
+      <Grid item sx={{ marginTop: "4px" }} data-testid="nps-banner-icon">
```

Comment:
I think you might have added this margin to make the icon center aligned vertically with the text. You should be able to add `display: flex` to the icon and have this render the same without needing to hard code the margin.